usecase/curriculum_usecase.go

```diff
@@ -0,0 +1,61 @@
+package usecase
```

Comment:
The use case is two separate words. It is better to use something like `use_case`

Reply:
#18